src/post/dto/response-post.dto.ts

```diff
@@ -0,0 +1,30 @@
+import { ApiProperty } from '@nestjs/swagger';
+
+export class ResponsePostDTO {
+    @ApiProperty()
+    id: string;
+
+    @ApiProperty()
+    userId: number;
+
+    @ApiProperty()
+    content: string;
+
+    @ApiProperty()
+    mediaUrl: string;
+
+    @ApiProperty()
+    likes: [];
+
+    @ApiProperty()
+    status: string;
+
+    @ApiProperty()
+    isDestroy: boolean;
```

Comment:
Don't need to return these fields: isDestroy, createdAt, updatedAt

Reply:
I have updated based on your review feedback. Please help me check again. If it's okay, please merge it for me.